local-experiment-config.yaml

```diff
@@ -0,0 +1,6 @@
+benchmarks:
```

Comment:
Actually, I don't want this in the root directory. Please move.

Reply:
Sounds good! Done.